libecl/src/ecl_kw.c

```diff
@@ -1240,9 +1259,11 @@ bool ecl_kw_fskip_data__( ecl_type_enum ecl_type , const int element_count , for
       const int block_count = element_count / blocksize + (element_count % blocksize == 0 ? 0 : 1);
 
       int element_size = ecl_util_get_sizeof_ctype(ecl_type );
-      if(ecl_type == ECL_CHAR_TYPE || ecl_type == ECL_MESS_TYPE) {
```

Comment:
Well - the element count of keywords of type `ECL_MESS`is *always* zero - i.e. there is no attached data; so I skipped it.